stack_helpers.sh

```diff
@@ -58,6 +58,7 @@ EOF
     CMAKE_OPTS="-DCMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_LIBDIR}"
     CMAKE_OPTS+=" -DTMPL_FILE=$tmpl_file -DVERSION=$version"
     [[ -n "${py_version:-}" ]] && CMAKE_OPTS+=" -DPYTHON_VERSION=$py_version"
+    CMAKE_OPTS+=" -DMAPL_ESMF_VERSION=${ESMF_VERSION:-}"
```

Comment:
There is nothing in `stack_helpers.sh` specific to any particular library.
Can this be kept out of `stack_helpers.sh`

Reply:
I have to protest for this hack.